sfs/mono/wfs.py

```diff
@@ -154,9 +154,13 @@ def _point(omega, x0, n0, xs, c=None):
 point_2d = _point
 
 
-def point_25d(omega, x0, n0, xs, xref=[0, 0, 0], c=None, omalias=None):
+def point_25d_legacy(omega, x0, n0, xs, xref=[0, 0, 0], c=None, omalias=None):
     r"""Driving function for 2.5-dimensional WFS for a virtual point source.
 
+    This driving function was used as standard `point_25d` in versions <=0.4.0.
```

Comment:
The word "standard" is not necessary here.

Reply:
I prefer using `` `point_25d()` `` (with parentheses).

Reply:
If you want to mention a version, you should use the `versionadded` or `versionchanged` directives, see https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html?highlight=deprecated#directive-versionadded.

In this case this could be something like

```rst
    .. versionadded:: 0.5

        This function was called `point_25d()` before. Blah blah ...
```

Reply:
fixed